### PR TITLE
Feature add mask for work hours field in reports form

### PR DESCRIPTION
**Pull request by dybi**
resolves: https://github.com/Code-Poets/sheetstorm/issues/315

### CODE REVIEW

File: employees/forms.py [hunk 1]
```diff
@@ -32,13 +32,13 @@ def __init__(self, queryset: QuerySet, *args: Any, **kwargs: Any) -> None:
...
     def format_value(self, value: Optional[str]) -> str:
-        if value is not None:
+        if value is not None and value != ":":
             return timedelta_to_string(parse_duration(value))
         return ""
 
 
 class DurationFieldForm(forms.DurationField):
-    widget = DurationInput
+    widget = DurationInput(attrs={"data-mask": "99:99", "placeholder": "H:MM"})
```
Comment (kbeker) [inline employees/forms.py]: What does it mean that this field has `99:99` mask?

Comment (dybi) [reply employees/forms.py]: https://dobsondev.com/2017/04/14/using-jquery-mask-to-mask-form-input/
> * ‘0’: {pattern: /\d/}
> * ‘9’: {pattern: /\d/, optional: true}

File: employees/templates/employees/report_list.html [hunk 2]
```diff
@@ -105,6 +105,9 @@ <h4><font color="red">{{ error }}</font></h4>
 {% endblock %}
 
 {% block extra_script %}
+    {{ form.media }}
+    <script src="//code.jquery.com/jquery-3.3.1.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery.mask/1.14.10/jquery.mask.js"></script>
```
Comment (rwrzesien) [inline employees/templates/employees/report_list.html]: Looks a little bit suspicious that we use two versions of the same library, maybe removing older version below will keep everything still working?

File: employees/forms.py (hunk 1 above)
Comment (rwrzesien) [reply employees/forms.py]: So shouldn't it be `90:00`? Which means that you should have at least one digit before `:` and two after. This way you can type in values like `12:00`, `8:00` and `0:30`.  Or maybe I have misunderstood this completely.

Comment (dybi) [reply employees/forms.py]: quite frankly, it must be `09:99` (checked empirically;)).

File: employees/templates/employees/report_list.html (hunk 2 above)
Comment (dybi) [reply employees/templates/employees/report_list.html]: good point. changed.

**Pull request merged**